src/Symfony/Component/Process/Process.php

```diff
@@ -340,7 +340,7 @@ public function start(callable $callback = null, array $env = [])
 
         $envPairs = [];
         foreach ($env as $k => $v) {
-            if (false !== $v && 'argc' !== $k && 'argv' !== $k) {
+            if (false !== $v && false === \in_array(strtolower($k), ['argc', 'argv'], true)) {
```

Comment:
I'd prefer not calling strtolower for that. Let's use this instead if yo don't mind:
`if (false !== $v && false === \in_array($k, ['argc', 'argv', 'ARGC', 'ARGV'], true)) {`